src/api/Gemfile

```diff
@@ -9,7 +9,7 @@ source 'https://rubygems.org'
 
 # Start the Rails application using src/api/Gemfile.next with `next rails s`
 if next?
-  gem 'rails', '~> 8.0'
+  gem 'rails', '~> 7.1.0'
```

Comment:
Why to 7.1.0? There is 7.1.5.1 already...

Reply:
The role of the [pessimistic operator ~>](https://thoughtbot.com/blog/rubys-pessimistic-operator) plays the trick here. It will take the highest-released version from the one declared to the upper one, in this case from:
`~> 7.1.0` means `>= 7.1.0` and `< 7.2`. So eventually we jump on `7.1.5.1` either way. I started by being conservative in the definition. I can fix it in the Gemfile, it will not affect the output anyway.